src/@types/smee-client/index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "smee-client" {
+  import EventSource = require('eventsource')
+
+  interface Options {
+      source?: string
+      target: string
+      logger?: console
```

Comment:
A small typo, it should be `Console`.

(Because `console` doesn't exist, and we're in a declaration file, it falls back to `any`.)